src/components/table/header.rs

```diff
@@ -28,6 +46,19 @@ pub fn table_header<K>(props: &TableHeaderProperties<K>) -> Html
 where
     K: Clone + Eq + 'static,
 {
+    let sort_by: UseStateHandle<Option<TableHeaderSortBy<K>>> = use_state_eq(|| None);
+    let on_sort_by = {
+        let sort_by = sort_by.clone();
+        Callback::from(move |val: Option<TableHeaderSortBy<K>>| {
+            sort_by.set(val);
+        })
+    };
```

Comment:
I know we use `Callback::from` a lot. But with function components and hook, we should try to use `use_callback` more often, as it creates a stable (equal) instance, cause less re-renders.

```suggestion
    let on_sort_by = use_callback(|val: Option<TableHeaderSortBy<K>>, sort_by|{
        sort_by.set(val);
    }, sort_by.clone());
```

The suggestion most likely needs some reformatting. But this will only create a new callback when the dependency (the `sort_by` handler) changes.

Reply:
Thanks for the suggestion. It has been applied to my last commit